lnrpc/lightning.proto

```diff
@@ -1531,6 +1531,9 @@ message Channel {
 
     // This is the confirmed / on-chain zero-conf SCID.
     uint64 zero_conf_confirmed_scid = 33;
+
+    // The alias of our peer.
```

Comment:
Should we be a bit more specific here? Because I think the peer can also have multiple SCID alias for the same channel. But this is the one that was originally negotiated? (<- actual question, I'm not fully up to date on the implementation of zero conf channels).

Reply:
As mentioned here https://github.com/lightningnetwork/lnd/pull/7366#issuecomment-1409013030 the `peerAlias` refers to the node configured peer alias name as supposed to the SCID alias. I'll point that out here.